contracts/Rewards/RewardsDistributor.sol

```diff
@@ -136,17 +136,17 @@ contract RewardsDistributor is
     /// @notice Emitted when a reward for contributor is updated
     event ContributorRewardsUpdated(address indexed contributor, uint256 rewardAccrued);
 
-    /// @notice Emitted when a reward token last rewarding block for supply is updated
-    event SupplyLastRewardingBlockUpdated(address indexed vToken, uint32 newBlock);
+    /// @notice Emitted when a reward token last rewarding slot (block or second) for supply is updated
+    event SupplyLastRewardingBlockUpdated(address indexed vToken, uint32 newBlockNumberOrTimestamp);
 
-    /// @notice Emitted when a reward token last rewarding block for borrow is updated
-    event BorrowLastRewardingBlockUpdated(address indexed vToken, uint32 newBlock);
+    /// @notice Emitted when a reward token last rewarding slot (block or second) for borrow is updated
+    event BorrowLastRewardingBlockUpdated(address indexed vToken, uint32 newBlockNumberOrTimestamp);
 
-    /// @notice Emitted when a reward token last rewarding block timestamp for supply is updated
-    event SupplyLastRewardingBlockTimestampUpdated(address indexed vToken, uint256 newBlock);
+    /// @notice Emitted when a reward token last rewarding slot (block or second) for supply is updated
+    event SupplyLastRewardingBlockTimestampUpdated(address indexed vToken, uint256 newBlockNumberOrTimestamp);
 
-    /// @notice Emitted when a reward token last rewarding block timestamp for borrow is updated
-    event BorrowLastRewardingBlockTimestampUpdated(address indexed vToken, uint256 newBlock);
+    /// @notice Emitted when a reward token last rewarding slot (block or second) for borrow is updated
+    event BorrowLastRewardingBlockTimestampUpdated(address indexed vToken, uint256 newBlockNumberOrTimestamp);
```

Comment:
These events don't accept block or timestamp, only one thing. For example, `SupplyLastRewardingBlockUpdated` is emitted in the `_setLastRewardingBlock` function, and it will include always the `supplyLastRewardingBlock` (that is a block). I would revert this change

Reply:
https://github.com/VenusProtocol/isolated-pools/pull/324/commits/25ed2b1ef2fc35b7f24628fdabadcd43d7037648